src/reducers/projects.js

```diff
@@ -83,6 +83,21 @@ export function reduceRoot(stateIn, action) {
             projects.get(currentProjectKey),
           );
         }
+      case 'FOCUS_LINE':
+        {
+          const currentProjectKey =
+            stateIn.getIn(['currentProject', 'projectKey']);
+
+          if (isNil(currentProjectKey)) {
+            return stateIn;
+          }
+
+          return projects.updateIn(
+            [currentProjectKey, 'hiddenUIComponents'],
+            hiddenUIComponents =>
+              hiddenUIComponents.remove(`editor.${action.payload.language}`),
+          );
+        }
```

Comment:
Not sure it matters too much, but would it be worth abstracting this and the logic in the `UNHIDE_COMPONENT` reducer into a single function? Not sure if the DRYness is worth the abstraction overhead in this case, though...